src/ProfiledAot/src/maui/App.xaml

```diff
@@ -4,33 +4,13 @@
              xmlns:local="clr-namespace:maui"
              x:Class="maui.App">
     <Application.Resources>
+        <!-- NOTE: if the template ever drastically changes, compare this to the template! -->
+        <ResourceDictionary Source="Resources/Styles.xaml" />
+        <!-- Everything to create in addition to the template -->
         <ResourceDictionary>
-
-            <Color x:Key="PrimaryColor">#512bdf</Color>
-            <Color x:Key="SecondaryColor">White</Color>
-            
-            
-            <Color x:Key="LightColor">White</Color>
-            <Color x:Key="DarkColor">Black</Color>
-
-            <Style TargetType="Page" ApplyToDerivedTypes="True">
-                <Setter Property="BackgroundColor"
-                        Value="{AppThemeBinding Light={StaticResource LightColor}, Dark={StaticResource DarkColor}}" />
-            </Style>
-
-            <Style TargetType="Label">
-                <Setter Property="TextColor" Value="{StaticResource PrimaryColor}" />
-                <!-- This line is the only change from the template, to use {OnPlatform} -->
-                <Setter Property="FontFamily" Value="{OnPlatform Default=OpenSansRegular, Android=OpenSansRegular}" />
-            </Style>
-
-            <Style TargetType="Button">
-                <Setter Property="TextColor" Value="{StaticResource SecondaryColor}" />
-                <Setter Property="FontFamily" Value="OpenSansRegular" />
-                <Setter Property="BackgroundColor" Value="{StaticResource PrimaryColor}" />
-                <Setter Property="Padding" Value="14,10" />
-            </Style>
-
+          <SolidColorBrush x:Key="ProfiledAot_AppThemeBinding" Color="{AppThemeBinding Default=Black}"/>
+          <SolidColorBrush x:Key="ProfiledAot_OnIdiom" Color="{OnIdiom Default=Black}"/>
+          <CollectionView x:Key="ProfiledAot_CollectionView" Margin="{AppThemeBinding Default=1,1,1,1}" />
```

Comment:
Why do we need an AppThemeBinding here?

I think it might make more "sense" if we use an OnIdiom here.

And then if we do that, do we need the 2nd Brush above?

Reply:
I think I can simplify to two lines:
```xml
<SolidColorBrush x:Key="ProfiledAot_AppThemeBinding_Color" Color="{AppThemeBinding Default=Black}"/>
<CollectionView x:Key="ProfiledAot_CollectionView_Idiom_Thickness" Margin="{OnIdiom Default==1,1,1,1}" />
```

Reply:
The profile comes out the same when I changed this.